backend/src/services/release.ts

```diff
@@ -237,9 +239,11 @@ export async function newReleaseComment(user: UserInterface, modelId: string, se
 export async function getModelReleases(
   user: UserInterface,
   modelId: string,
+  q?: string,
 ): Promise<Array<ReleaseDoc & { model: ModelInterface; files: FileInterface[] }>> {
+  const query = q === undefined ? { modelId } : getQuerySyntax(q, modelId)
   const results = await Release.aggregate()
-    .match({ modelId })
+    .match(query!)
```

Comment:
By using `!` we're telling TS that this can't possibly be `undefined`/`null`. This is very rarely the correct way to manage types and is likely indicative of a flaw in your `getQuerySyntax()` logic.

I'd suggest removing this assertion and seeing where `undefined`/`null` are possibilities in earlier logic. You should fix it there instead of asserting here.

Reply:
Turns out I didn't even need it anymore so just removed